### ISSUE TITLE
ShortCustomUuid::parse_str only seems to work with 58-character alphabets

### ISSUE THREAD

**Issue by kmdreko** [radim10/short-uuid#4]
Trying to parse a short UUID with a custom alphabet that isn't 58-characters doesn't seem to work. This code below demonstrates a failure to parse when using `COOKIE_BASE_90`:

```
use short_uuid::{CustomTranslator, ShortUuidCustom, COOKIE_BASE_90};

fn main() {
    let translator = CustomTranslator::new(COOKIE_BASE_90).unwrap();
    let orig_uuid_str = "420e04de-4f06-4e3b-8718-9f6ede92ea9e";
    let short = ShortUuidCustom::from_uuid_str(orig_uuid_str, &translator).unwrap();
    let short_uuid_str = short.to_string();
    let short = ShortUuidCustom::parse_str(&short_uuid_str, &translator).unwrap();
}
```

```
thread 'main' panicked at src/main.rs:8:74:
called `Result::unwrap()` on an `Err` value: InvalidShortUuid
```

**Comment by radim10** [comment]
Thanks, already fixed with PR #5.

**Comment by radim10** [comment]
Thanks, already fixed with PR #5.
